games/minimal/mods/experimental/mod.conf

```diff
@@ -1,3 +1,2 @@
 name = experimental
-description = Minimal mod to test features
-depends = default, stairs
+description = Chaotic mod containing unstructured tests for testing out engine features. The features in this mod should be moved to other mods.
```

Comment:
I see three tests in here: node callbacks, node meta and particles. Node callbacks & meta could be merged to a node functionality testing mod?